l10n_it_vat_registries/models/account.py

```diff
@@ -16,6 +16,11 @@ class AccountTax(models.Model):
     parent_tax_ids = fields.Many2many(
         'account.tax', 'account_tax_filiation_rel', 'child_tax', 'parent_tax',
         string='Parent Taxes')
+    
+    cee_type = fields.Selection([
+            ('sale', 'Sale'),
+            ('purchase', 'Purchase')
+            ], 'Cee Type')
```

Comment:
Ci siamo rese conto che cee_type è un nome infelice. 
Nel caso di integrazione IVA,  in fattura vengono create due movimenti iva, ma nella stampa del registro IVA vengono visualizzate entrambe con segni opposti. Con questo campo indichiamo su quale registro IVA deve essere visualizzata e conteggiata.